dev_tools/scripts/mongo_test.sh

```diff
@@ -0,0 +1,49 @@
+#!/bin/bash
+
+num=100
+
+if [[ -n $1 ]]; then
```

Comment:
This obviously works, but I'm always more comfortable with protecting variables with quotes.

Reply:
Currently bashing my head against testing with TLS enabled.